src/Oscar.jl

```diff
@@ -156,7 +156,7 @@ function __init__()
 
   add_assertion_scope(:ZZLatWithIsom)
   add_verbosity_scope(:ZZLatWithIsom)
-  
```

Comment:
if possible, please drop these kinds of changes from your PR again, as they will just lead to unrelated conflicts without real benefit.